pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1155,6 +1157,10 @@ func (c *Cloud) Initialize(clientBuilder controller.ControllerClientBuilder) {
 	c.eventBroadcaster.StartLogging(glog.Infof)
 	c.eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: c.kubeClient.CoreV1().Events("")})
 	c.eventRecorder = c.eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "aws-cloud-provider"})
+
+	registerMetricsOnce.Do(func() {
+		registerMetrics(c)
+	})
```

Comment:
But now this will mean that metrics unrelated to this change are not registered on node. Are you sure - other metrics are not being emitted at all on node? I would think - we are at least calling describeInstance on the node. I am not 100% sure, but we should be sure that - emitting a unregistered metric on node, does not causes crashes or anything.

I think - we have no choice but other than to create new function that registers these metrics only for controller-manmagr.

Reply:
agree, added `registerControllerMetrics` to be safe